### PR TITLE
feat: CORE-1155 WhatsApp Messaging Feature

### PR DESCRIPTION
**Pull request by reinlabares**
CORE-1155 Documentation for WhatsApp Messaging Feature

### CODE REVIEW

**Review by aleksandar-milisic — APPROVED** [review]

**Comment by richardfontein** [comment]
Thanks for your contribution @reinlabares.

We've recently merged a fix into main which deals with some Spectral style linting issues in CI - you may need to fix any style errors that present after merge, these will be annotated on the commit.

Could I please also ask that you modify the commit prefix to `feat:` per the [conventional commits spec](https://www.conventionalcommits.org/en/v1.0.0/) - we'll treat this as a new feature in the API requiring a minor version bump, as far as our downstream SDKs are consumed.

File: openapi.yaml [hunk 1]
```diff
@@ -10613,6 +10616,294 @@ components:
...
+            type: icebreaker
+            name: account_update_example_image_p
+            content:
+              header:
+                image:
+                  link: 'http://placekitten.com/200/300'
+              body:
+                - text: xxxxx
+                - text: yyyyy
+                - text: zzzzz
```
Comment (richardfontein) [inline openapi.yaml:10644]: Please use more meaningful examples - they're use to populate examples that developers will use to try out the API, and it's important for their developer experience to have high quality examples to enable them to get started quickly and with content that demonstrates how they might use the API.

For example:
```
to: '61430933333'
subject: Account Update
whatsapp:
  type: icebreaker
  name: account_update_example_image_p
  content:
    header:
      image:
        link: 'http://placekitten.com/200/300'
    body:
      - text: Your account has been updated.
      - text: If you did not action this update, please change your password immediately.
      - text: Thank you, Whispir
```

**Review by richardfontein — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 1 above)
Comment (reinlabares) [reply openapi.yaml]: I have already modified the examples and changed the title from **chore** to **feat**. Thank you

**Comment by reinlabares** [comment]
A new PR was created in place of this

**Pull request closed without merging**